travis/test_server.py

```diff
@@ -302,7 +298,13 @@ def main(argv=None):
             test_loghandler = 'openerp.tools.yaml_import:DEBUG'
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
     server_path = get_server_path(odoo_full, odoo_version, travis_home)
-    script_name = get_server_script(odoo_version)
+    # Normalize on v10 odoo-bin command
+    try:
+        shutil.copy(
+            os.path.join(server_path, 'openerp-server'),
+            os.path.join(server_path, 'odoo-bin'))
```

Comment:
If you men WIP code = working directory, then I need to correct you. The test bed is a copied (or rsynced) special folder structure in the most common case here... From this copy, nothing is fed back, never. You could call it prep bed, for "preparation". This has almost the word "stateful" inherent. If for any improbable chance the working directory folder structure happens to coincide with the test bed folder structure (in case of adding enterprise it cant, for example), we should rather force the use of a testbed, there...